RDSC-4633 How to perform HA failover (#2818)
redis/docs: content/integrate/redis-data-integration/installation/ha-test.md

---

## Setup
1. Ensure that RDI is up and running on both primary and secondary nodes.
   Run the following command and verify and that each instance should show healthy and running `rdi-api` and `rdi-operator` pods.
```
kubectl -n rdi get pods

# Example output:
NAME                                   READY   STATUS      RESTARTS      AGE
collector-api-577d95bfd8-5wbg6         1/1     Running     0             12m
collector-source-95f45bcf7-vwn5l       1/1     Running     0             12m
fluentd-zq2lc                          1/1     Running     0             72m
logrotate-29530445-j729x               0/1     Completed   0             14m
logrotate-29530450-dprr2               0/1     Completed   0             9m40s
logrotate-29530455-mfmzw               0/1     Completed   0             4m40s
processor-f66655469-h7nw2              1/1     Running     0             12m
rdi-api-f75df6796-qwqjw                1/1     Running     0             72m
rdi-metrics-exporter-d57cdf8c8-wjzb5   1/1     Running     0             72m
rdi-operator-7f7f6c7dfd-5qmjd          1/1     Running     0             71m
rdi-reloader-77df5f7854-lwmvz          1/1     Running     0             71m
```

> 2. Identify the leader node - this is the one that has a running `collector-source` pod

Reviewer: ```suggestion
2. Identify the leader node - this is the one that has a running `collector-source` pod.
```

## Performing the HA Failover Testing

To perform HA, you can simulate a connection failure between the leader and the RDI database by blocking the network traffic. You can do this by running the following command on the leader node:

> 1. Identify the database IP (if you are using it with hostname):

Reviewer: ```suggestion
1. Identify the database IP (replace `<hostname>` with your own hostname):
```

Author: ```suggestion
1. Identify the RDI database IP (replace `<hostname>` with your own hostname):
```
